backend/tests/Functional/Service/SecurityAlertServiceTest.php

```
@@ -0,0 +1,232 @@
+            'roles_after' => ['ROLE_ADMIN', 'ROLE_USER'],
+            // Everything below is the sort of thing a caller might attach to
+            // the log record. None of it belongs in an inbox.
+            'password' => 'should-never-travel',
+            'access_token' => 'sl.B0123456789',
+            'comic_title' => 'An Explicit Title',
+            'invitation_token' => 'abcdef123456',
+        ]);
+
+        $body = $mailer->messages[0]->getHtmlBody();
+
+        self::assertStringContainsString('security.admin_role.changed', $body);
```

PHPStan (2.2.7)

[error] Call to an undefined static method App\Tests\Functional\Service\SecurityAlertServiceTest::assertStringContainsString().


(staticMethod.notFound)